### PR TITLE
[FIX] account: reconcile from journal items bugfix

### PR DESCRIPTION
**Pull request by william-andre**
OPW 1919727

when you go in the journal items, filter on unreconciled lines, pick just one and click on 'Action > reconcile', the manual reconciliation widget opens but you directly get the rainbowman and you can't do anything. It's not possible to reconcile the selected entry with a writeoff (to clear a customer account, for example)
remove that check in the manual reconciliation widget, in case we have active_ids and active_model == 'account.move.line'

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
@william-andre I'd rather see, at line 266:

``` python
if aml_ids:
    #check all selected lines have the same company or raise an error
    ...
    #check all selected lines have the same account or raise an error
    ...
    #check that account is set as reconciliable or raise an error
    ...

#go on with the current SQL query untouched
```

as it would be more explicit and also would fulfill the point of the spec: ` implement real error message when selected lines don't have the same company or account (currently it's displaying a rainbowman): see error messages in 'reconcile()' in account_move.py`

File: addons/account/models/reconciliation_widget.py [hunk 1]
```diff
@@ -286,22 +290,23 @@ def get_data_for_manual_reconciliation(self, res_type, res_ids=None, account_typ
                         {4}
                         {5}
                         AND l.company_id = {6}
-                        AND EXISTS (
+                        AND ({7} OR (
```
Comment (qdp-odoo) [inline addons/account/models/reconciliation_widget.py]: add an in-line comment to depict the use case and what this aml_ids argument does

**Comment by william-andre** [comment]
@qdp-odoo

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**